docs/generator.go

```diff
@@ -2,42 +2,86 @@ package main
 
 import (
 	"encoding/json"
+	"flag"
 	"fmt"
 	"io/ioutil"
 	"log"
 	"os"
+	"strings"
 	"text/template"
+
+	"github.com/tidwall/gjson"
```

Comment:
As the gjson package is at vendor folder, we will have to make the desired changes here.

Reply:
That's the beauty of the vendor folder! It's detected in addition to the GOPATH :P